platform/launch.sh

```diff
@@ -26,6 +26,16 @@ declare cygwin=false
 declare darwin=false
 declare linux=false
 declare dc_exec="docker-compose up"
+declare nightly=""
```

Comment:
isn't it a `version` variable instead?

how would you manage compose file per version ?
How to go from a 1.x to 3.x for example ?

Reply:
Currently, there is only the capability to run either latest either nightly but I can prepare a more sophisticated solution to start any version of the platform.

Reply:
That's ok for this PR. We have to follow-up on this.